OpenHPL/Waterway/AirCushionSurgeTank.mo

```diff
@@ -0,0 +1,79 @@
+within OpenHPL.Waterway;
+model AirCushionSurgeTank "Model of air-cushion surge tank"
+  outer Parameters para "using standard class with constants";
+  extends OpenHPL.Icons.Surge;
+  import Modelica.Constants.pi;
+  //// geometrical parameters of air-cushion surge tank
+  parameter Modelica.SIunits.Height H = 10.5 "Vertical component of the length of the surge shaft" annotation (
+    Dialog(group = "Geometry"));
+  parameter Modelica.SIunits.Length L = 12.12 "Length of the surge shaft" annotation (
+    Dialog(group = "Geometry"));
+  parameter Modelica.SIunits.Diameter D = 50.47 "Diameter of the surge shaft" annotation (
+    Dialog(group = "Geometry"));
+  parameter Modelica.SIunits.Height eps = para.eps "Pipe roughness height" annotation (
+    Dialog(group = "Geometry"));
+  //// condition for steady state
+  parameter Boolean SteadyState = para.Steady "if true - starts from Steady State" annotation (
+    Dialog(group = "Initialization"));
+  //// steady state values for flow rate and water level in surge tank
+  parameter Modelica.SIunits.VolumeFlowRate V_dot0 = 0 "Initial flow rate in the surge tank" annotation (
+    Dialog(group = "Initialization"));
+  parameter Modelica.SIunits.Height h_0 = 2 "Initial water height in the surge tank" annotation (
+    Dialog(group = "Initialization"));
+  input Modelica.SIunits.Pressure p_c0 = 4*para.p_a "Initial pressure of air-cushion inside the surge tank" annotation (
+    Dialog(group = "Geometry"));
+  //// possible parameters for temperature variation. Not finished...
+  //parameter Boolean TempUse = para.TempUse "If checked - the water temperature is not constant" annotation (Dialog(group = "Initialization"));
+  //parameter Modelica.SIunits.Temperature T_i = para.T_i "Initial water temperature in the pipe" annotation (Dialog(group = "Initialization", enable = TempUse));
+  //// variables
+  Modelica.SIunits.Mass m "water mass";
+  Modelica.SIunits.Momentum M "water momuntum";
+  Modelica.SIunits.Force M_dot "difference in influent and effulent momentum";
+  Modelica.SIunits.Force F "total force acting in the surge tank";
+  Modelica.SIunits.Area A = (pi*D ^ 2) / 4 "cross sectional area of the surge tank";
+  Modelica.SIunits.Length l = h / cos_theta "length of water in the surge tank";
+  Real cos_theta = H / L "slope ratio";
+  Modelica.SIunits.Velocity v "water velocity";
+  Modelica.SIunits.Force F_p "pressure force";
+  Modelica.SIunits.Force F_f "friction force";
+  Modelica.SIunits.Force F_g "friction force";
+  Modelica.SIunits.Pressure p_c "air-cushion force";
```

Comment:
Convention is that description strings should start with an upper case. E.g., `"Water mass"`.